### ISSUE TITLE
splits and :tabedit do not work nicely with each other / do not have vim behavior 

### ISSUE THREAD

**Issue by jerabaul29** [VSCodeVim/Vim#4746]
It looks like ```:tabedit``` and the ```split```s (```:split``` and ```:vsplit```) do not work so nicely together.

If I am right:

- in vim, ```:tabedit``` opens tabs, and then within the tab it is possible to open as many splits as wanted. By doing ```gt``` or ```gT```, one moves between tabs, each containing all the corresponding splits. This is convenient, as I can have a tab for fixing something, another for working on another issue, etc.

By contrast here:

- using ```gt/T``` for moving between tabs move only the tab in the current split. I.e., we have a split of tabs, rather than a tab of splits.

- combined with https://github.com/VSCodeVim/Vim/issues/4745 I think this leads to behaviors that are quite unlike vim (?).

**Comment by J-Fields** [comment]
Yeah this has been brought up before. I'm not really sure how we emulate splits-within-tabs when VSCode only supports tabs-within-splits natively. This is probably an instance of where rigidly emulating vim's behavior would violate the principle of least astonishment.

**Comment by jerabaul29** [comment]
Wonder of, in case this violated least astonishment principle, this could be a setting of the plugin on the plugin page? Vim mode Vs. VSC mode for tabs and splits or something like that?
